src/core/toga/widgets/table.py

```diff
@@ -100,60 +87,61 @@ class Table(Widget):
     Args:
         headings (``list`` of ``str``): The list of headings for the table.
         id (str): An identifier for this widget.
+        data (``list`` of ``list`): The data to be displayed on the table.
         style (:class:`colosseum.CSSNode`): An optional style object.
-            If no style is provided then a new one will be created for the widget.
+            If no style is provided` then a new one will be created for the widget.
+        on_select(``callable``): A function to be invoked on selecting a row of the table.
         factory (:obj:`module`): A python module that is capable to return a
             implementation of this class with the same name. (optional & normally not needed)
+
+    Examples:
+        >>> headings = ['Head 1', 'Head 2', 'Head 3']
+        >>> data = [['item 1', 'item 2', 'item3'],
+        >>>         ['item 1', 'item 2', 'item3']]
```

Comment:
Although both will work, this should be a list of tuples - the inner items have position specific type.